### PR TITLE
Add portfolio link

### PR DESCRIPTION
**Pull request by eduardconstantin**
Added my portfolio link to the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 A list of developer portfolios for your inspiration
 
-Have you built a portfolio? Are you proud of it?! Open a [PR](./CONTRIBUTING.md) to this repo and let's showcase your work! Refer to the [CONTRIBUTING](./CONTRIBUTING.md) file for direction.
+Have you built a portfolio? Are you proud of it?! Open a [PR](./CONTRIBUTING.md) to this repo and let's showcase your
+work! Refer to the [CONTRIBUTING](./CONTRIBUTING.md) file for direction.
```
Comment (eclectic-coding) [inline README.md:6]: Please remove the edits to this paragraph

File: README.md [hunk 2]
```diff
@@ -11,7 +12,9 @@ Hopefully this repo can serve as a source of inspiration for your portfolio!
 
 ---
 
-**Jump to:** [A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f) | [G](#g) | [H](#h) | [I](#i) | [J](#j) | [K](#k) | [L](#l) | [M](#m) | [N](#n) | [O](#o) | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | [Y](#y) | [Z](#z)
+**Jump to:** [A](#a) | [B](#b) | [C](#c) | [D](#d) | [E](#e) | [F](#f) | [G](#g) | [H](#h) | [I](#i) | [J](#j) | [K](#k)
+| [L](#l) | [M](#m) | [N](#n) | [O](#o) | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w)
+| [Y](#y) | [Z](#z)
```
Comment (eclectic-coding) [inline README.md:17]: Please remove the edits to this paragraph

**Review by eclectic-coding — COMMENTED** [review]
Make these changes requested and I will merge your site to the list. Thanks for your interest.

**Comment by eduardconstantin** [comment]
Sorry, I use format on save.
Thank you!

**Pull request merged**